gcecreateimg

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python3
+
+# Copyright 2025 SUSE LLC
+#
+# This file is part of gceimgutils
+#
+# gceimgutils is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# gceimgutils is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with gcecreateimg. If not, see <http://www.gnu.org/licenses/>.
+
+import argparse
+import json
+import logging
+import os
+import sys
+
+from gceimgutils.gceutils import image_to_dict, get_logger, get_version
+from gceimgutils.gcecreateimg import GCECreateImage
+
+# Set up command line argument parsing
+argparse = argparse.ArgumentParser(description='Insert image in GCE')
+argparse.add_argument(
+    '-c', '--credentials-file',
+    dest='credentials_file_path',
+    help='Path to the service account credentials file',
+    metavar='CREDENTIALS_FILE'
+)
+argparse.add_argument(
+    '-p', '--project',
+    dest='project_name',
+    help='Project to use for image creation',
+    metavar='PROJECT_NAME',
+    required=True
+)
+argparse.add_argument(
+    '--image-name',
+    dest='image_name',
+    help='The image name of the image to be created',
+    required=True
+)
+argparse.add_argument(
+    '--image-description',
+    dest='image_description',
+    help='The image description of the image to be created',
+    required=True
+)
+argparse.add_argument(
+    '--family',
+    dest='family',
+    help='The image family of the image to be created',
+    required=True
+)
+argparse.add_argument(
+    '--bucket-name',
+    dest='bucket_name',
+    help='The bucket name where the image tarball is located',
+    required=True
+)
+argparse.add_argument(
+    '--object-name',
+    dest='object_name',
+    help='The object name of the image tarball',
+    required=True
+)
+argparse.add_argument(
+    '--architecture',
+    dest='architecture',
+    help='The architecture for the compute image',
```

Comment:
Should be a choice, we only want to allow `x86_64` and `arm64`

Reply:
Updated